test/examples/utils/plotting_helpers.jl

```diff
@@ -102,7 +102,7 @@ function plot_spectra!(sol, p; file_name = "test_spectra.png", logxrange = (-15,
         plt[i] = plot()
         for j in 1:Ndist
             ind_rng = get_dist_moments_ind_range(p.NProgMoms, j)
-            update_dist_from_moments!(p.pdists[j], moments[t_ind[i], ind_rng])
+            update_dist_from_moments(p.pdists[j], moments[t_ind[i], ind_rng])
```

Comment:
```suggestion
            p.pdists[j] = update_dist_from_moments(p.pdists[j], moments[t_ind[i], ind_rng])
```
Does this need to return the dist?

Reply:
I rewrote `update_dist_from_moments` to return a new struct instead of modifying the input struct, so it does need to return the dist for now.